archlinux-minimal: boost_cropped: error: parameter declared 'auto'

```
  [boost_cropped-1.88.0]   [spkg-install] /usr/include/unicode/localpointer.h:561:26: error: parameter declared 'auto'
  [boost_cropped-1.88.0]   [spkg-install]   561 | template <typename Type, auto closeFunction>
  [boost_cropped-1.88.0]   [spkg-install]       |                          ^~~~
  [boost_cropped-1.88.0]   [spkg-install] /usr/include/unicode/localpointer.h:573:76: error: template argument 2 is invalid [-Wtemplate-body]
  [boost_cropped-1.88.0]   [spkg-install]   573 |     explicit LocalOpenPointer(std::unique_ptr<Type, decltype(closeFunction)> &&p)
  [boost_cropped-1.88.0]   [spkg-install]       |                                                                            ^
  [boost_cropped-1.88.0]   [spkg-install] /usr/include/unicode/localpointer.h:583:78: error: template argument 2 is invalid [-Wtemplate-body]
  [boost_cropped-1.88.0]   [spkg-install]   583 |     LocalOpenPointer &operator=(std::unique_ptr<Type, decltype(closeFunction)> &&p) {
  [boost_cropped-1.88.0]   [spkg-install]       |                                                                              ^
  [boost_cropped-1.88.0]   [spkg-install] /usr/include/unicode/localpointer.h:599:59: error: template argument 2 is invalid [-Wtemplate-body]
  [boost_cropped-1.88.0]   [spkg-install]   599 |     operator std::unique_ptr<Type, decltype(closeFunction)> () && {
  [boost_cropped-1.88.0]   [spkg-install]       |                                                           ^
  [boost_cropped-1.88.0]   [spkg-install] /usr/include/unicode/uenum.h:69:1: note: invalid template non-type parameter
  [boost_cropped-1.88.0]   [spkg-install]    69 | U_DEFINE_LOCAL_OPEN_POINTER(LocalUEnumerationPointer, UEnumeration, uenum_close);
  [boost_cropped-1.88.0]   [spkg-install]       | ^~~~~~~~~~~~~~~~~~~~~~~~~~~
  [boost_cropped-1.88.0]   [spkg-install] /usr/include/unicode/ucnv.h:597:1: note: invalid template non-type parameter
  [boost_cropped-1.88.0]   [spkg-install]   597 | U_DEFINE_LOCAL_OPEN_POINTER(LocalUConverterPointer, UConverter, ucnv_close);
  [boost_cropped-1.88.0]   [spkg-install]       | ^~~~~~~~~~~~~~~~~~~~~~~~~~~
  [boost_cropped-1.88.0]   [spkg-install] In file included from /usr/include/unicode/unistr.h:37,
  [boost_cropped-1.88.0]   [spkg-install]                  from /sage/local/var/tmp/sage/build/boost_cropped-1.88.0/src/libs/locale/src/encoding/../icu/uconv.hpp:18:
  [boost_cropped-1.88.0]   [spkg-install] /usr/include/unicode/char16ptr.h:317:10: error: 'is_convertible_v' is not a member of 'std'; did you mean 'is_convertible'?
  [boost_cropped-1.88.0]   [spkg-install]   317 |     std::is_convertible_v<T, std::u16string_view>
  [boost_cropped-1.88.0]   [spkg-install]       |          ^~~~~~~~~~~~~~~~
  [boost_cropped-1.88.0]   [spkg-install]       |          is_convertible
  [boost_cropped-1.88.0]   [spkg-install] /usr/include/unicode/char16ptr.h:317:28: error: expected primary-expression before ',' token
  [boost_cropped-1.88.0]   [spkg-install]   317 |     std::is_convertible_v<T, std::u16string_view>
  [boost_cropped-1.88.0]   [spkg-install]       |                            ^
  [boost_cropped-1.88.0]   [spkg-install] /usr/include/unicode/char16ptr.h:331:13: error: 'u16string_view' in namespace 'std' does not name a type; did you mean 'u16string'?
  [boost_cropped-1.88.0]   [spkg-install]   331 | inline std::u16string_view toU16StringView(std::u16string_view sv) { return sv; }
  [boost_cropped-1.88.0]   [spkg-install]       |             ^~~~~~~~~~~~~~
  [boost_cropped-1.88.0]   [spkg-install]       |             u16string
  [boost_cropped-1.88.0]   [spkg-install] /usr/include/unicode/char16ptr.h:339:13: error: 'u16string_view' in namespace 'std' does not name a type; did you mean 'u16string'?
```

--- a/build/pkgs/boost_cropped/spkg-install.in
+++ b/build/pkgs/boost_cropped/spkg-install.in
@@ -11,6 +11,8 @@ set -ex
 # process:
 #     [boost_cropped-1.88.0]   [spkg-install] C:/a/passagemath/passagemath/sage-local/var/tmp/sage/build/boost_cropped-1.88.0/src/libs/process/include/boost/process/v2/detail/environment_win.hpp:43:8: error: constexpr function's return type 'void' is not a literal type
 #    [boost_cropped-1.88.0]   [spkg-install]    43 |   void assign(char_type& c1, const char_type& c2) BOOST_NOEXCEPT
+# locale:
+#     see https://github.com/passagemath/passagemath/issues/1227
 #
 # BOOST_STACKTRACE_LIBCXX_RUNTIME_MAY_CAUSE_MEMORY_LEAK:
 # While building 32-bit Linux wheels:
@@ -22,7 +24,7 @@ sdh_cmake -GNinja \
       -DBUILD_STATIC_LIBS=OFF \
       -DBUILD_SHARED_LIBS=ON \
       -DBOOST_INSTALL_LAYOUT=tagged \
-      -DBOOST_EXCLUDE_LIBRARIES="cobalt;process" \
+      -DBOOST_EXCLUDE_LIBRARIES="cobalt;process;locale" \
       -DBOOST_STACKTRACE_ENABLE_FROM_EXCEPTION=0 \
       .
 cmake --build .

